packages/runtime-dom/src/components/Transition.ts

```diff
@@ -99,7 +100,10 @@ export function resolveTransitionProps({
     done && done()
     // reset enter class
     if (appear) {
-      ;[enterFromClass, enterActiveClass, enterToClass] = originEnterClass
+      transitionNumber--
+      if (transitionNumber === 0) {
+        ;[enterFromClass, enterActiveClass, enterToClass] = originEnterClass
+      }
```

Comment:
For `TransitionGroup ` has more than one child nodes. Need reset class when last node transition end.
This solution can be work.Maybe has more better solution, but I just has this one.